.sequelizerc

```diff
@@ -0,0 +1,8 @@
+require('babel-register');
+const path = require('path');
+
+module.exports = {
```

Comment:
Check well the path / based on your folder strcuture

Reply:
I think the path is well set, isn't it?